src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -295,114 +589,220 @@ td.main, td.menu {
-    height: 30px;
-    display: none;
+#twig-dump .status-warning {
+    background: rgba(240, 181, 24, 0.3);
 }
-.sf-profiler-timeline .legends {
+#twig-dump .status-ok {
+    background: rgba(100, 189, 99, 0.2);
+}
+
+table.logs .metadata {
+    display: block;
     font-size: 12px;
-    line-height: 1.5em;
+    padding-top: 4px;
 }
-.sf-profiler-timeline .legends span {
-    border-left-width: 10px;
-    border-left-style: solid;
-    padding: 0 10px 0 5px;
+
+.status-ok {
+    background: rgba(100, 189, 99, 0.2);
+    background: rgba(94, 151, 110, 0.2);
 }
-.sf-profiler-timeline canvas {
-    border: 1px solid #999;
-    border-width: 1px 0;
+.status-warning {
+    background: rgba(240, 181, 24, 0.2);
+}
+.status-error {
+    background: rgba(204, 0, 0, 0.15);
+    background: rgba(176, 65, 62, 0.1);
+}
+.status-ok td,
+.status-warning td,
+.status-error td
+ {
+    background: transparent;
+}
+
+#search-results td {
+    vertical-align: middle;
 }
-.collapsed-menu-parents #resume,
-.collapsed-menu-parents #collector-content {
-    margin-left: 60px !important;
+#search-results .status-warning td.bold  {
+    color: #BC8034;
 }
-.collapsed-menu {
-    width: 60px !important;
+#search-results .status-error td.bold  {
+    color: #B0413E;
 }
-.collapsed-menu span.label > :not(.icon) {
-    display: none !important;
+
+.sql-queries th span {
+    color: #999;
 }
-.collapsed-menu #minimizePanelIcon {
-    transform: rotate(180deg);
+.sql-section
+.sql-actions {
+    margin-top: 4px;
 }
+.sql-actions a {
+    font-size: 12px;
+    margin-right: 1em;
+    text-decoration: underline;
+}
+.sql-actions a:hover {
+    text-decoration: none;
+}
+.highlight-sql {
+    font-family: monospace;
+    font-size: 13px;
+    line-height: 1.5;
+}
+.sql-parameters {
+    margin: 4px 0;
+}
+.sql-runnable {
+    background: #F5F5F5;
+    display: none; /* initially they are hidden */
+    margin: .5em 0;
+    padding: 1em;
+}
+
+/*.highlight-sql { background: #444; padding: 1em; }
+.sql-keyword   { color: #FF8400; }
+.sql-word      { color: #FFFFFF; }
+.sql-variable  { color: #FFFFFF; }
+.sql-symbol    { color: #FFFFFF; }
+.sql-comment   { color: #A0A0A0; }
+.sql-backtick  { color: #FF8400; }
+.sql-string    { color: #56DB3A; }
+.sql-number    { color: #71C8F4; }
+.sql-error     { color: #CC0000; }
+*/
+
+.sql-keyword   { color: #AA2063; font-weight: bold; }
```

Comment:
IMO, highlighting CSS rules should not be specific to SQL. what if another panel is highlighting something else than SQL ?